woocommerce/woocommerce-android
# Issue/5341 capture terminal endpoint active plugin

Merge instructions:

1. Review this PR
2. Make sure "Connection tokens endpoint" endpoint for active plugin https://github.com/woocommerce/woocommerce-android/pull/5538 is merged
3. Make sure capture payment endpoint for active plugin https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2232 is merged
4. Update fluxc hash in this PR
5. Remove "Do not merge" label
6. Merge this PR

Closes: #5341 


### Description

This PR updates WCAndroid so it's compatible with fluxC after introduction of `/wc_stripe/orders/<order_id>/capture_terminal_payment` endpoint - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2232

### Testing instructions

1. Install the newest version of the Stripe Extension plugin to your store (the version hasn't been released yet, you'll need to build it locally)
2. Make sure WCPay is not active in your store
3. Open the App
4. Navigate to the Order detail screen which is eligible for IPP
5. Tap on the "Collect Payment" button
6. Check in the debugger that `/wc_stripe/orders/<order_id>/capture_terminal_payment` endpoint is invoked. You can use a flipper or a similar tool for this as well.
